frontend/src/datatypes/enums.types.ts

```diff
@@ -0,0 +1,22 @@
+export enum Bedtypes {
```

Comment:
You have these enums twice. Once here and once in `frontend/src/datatypes/ListView.types.ts` 😉 Please remove one of the definitions 😉

Reply:
Fixed in 53a5c87